### PR TITLE
Add get help FAQ's

### PR DESCRIPTION
**Pull request by LukeSeers**

### CODE REVIEW

**Comment by davegson** [comment]
Good work 👏, I'll get starting on adding questions and re-texting some stuff

**Comment by davegson** [comment]
implemented these properly via a .yml so we can easily add / update them. Please review @LukeSeers

**Comment by davegson** [comment]
and @dhaavi, please review technical accuracy 😉

File: _data/faq.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+      url: https://safing.io/assets/img/spn-labeled/spn-community.png
+  - q: Does the SPN route connections on a per app basis?
+    a: No, it routes every connection individually. So as an example, when you open several websites in different tabs in your browser, every connection will be calculated individually.
+  - q: Am I totally safe from tracking with the SPN? What do service providers see when connecting through the SPN?
+    a: No. This is just one piece of the puzzle. The SPN only protects your IP-address from exposure, but there are still many other ways services can track and identify you. These include fingerprinting (mostly in browsers), authentication (when logged in to a service), mouse movements (malicious javascripts), etc...
+  - q: Can services still identify me through other methods?
+    a: Yes, this is just one piece of the puzzle. There are multiple ways services track you, like browser fingerprinting, malicious javascripts, etc... And if you log into services, such as your Google account, they will mostly still know it is you, even though you protect your IP address.
```
Comment (dhaavi) [inline _data/faq.yml:24]: Accurate, but somewhat a duplicate.
Suggestions only.

File: _data/faq.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  - q: Does the SPN only protect my browser?
+    a: The SPN reroutes all your computers connections, including those of your browser.
+  - q: Can I safely log in to sites via the SPN? Like my bank account etc?
+    a: Yes. All serious websites use secure https connections, which encrypts the content of your connection (like login credentials) before the Portmaster wraps them in onion encryption. So no ISP architecture or SPN exit node will be able to manipulate that data. It is protected by https all the way to the destination server.
```
Comment (dhaavi) [inline _data/faq.yml:28]: We could add here that unencrypted connections (ie. without HTTPS) will exit the network at special nodes with extra protection to ensure nothing funky happens.
We are not doing this yet, but will until the end of the year.

File: _data/faq.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+  - q: What platforms will you support?
+    a: Since the SPN is integrated into the Portmaster, for our the pre-alpha stage we will only support Windows and Linux. Mac and Mobile are planned, but are further down the road.
+  - q: Is there a free trial version for people interested in the SPN?
+    a: No. Providing the network costs us money, so we charge accordingly. However, the privacy filter and DNS Resolver, which add a great value to your privacy, are and will remain free to use.
+portmaster:
+  - q: Since the Portmaster can block ads and trackers, do I still need an ad-blocker for my browser?
+    a: Yes, we highly recommend to continue using adblockers, such as uBlock Origin, uMatrix, or similar. They give you more fine grained control over domains. iE you can allow certain domains to allow css files & images while blocking cookies & javascripts. The Portmaster cannot make this distinction.
+  - q: What platforms do you support?
+    a: Currently we support Windows and Linux. Mac and Mobile are planned, but are further down the road.
```
Comment (dhaavi) [inline _data/faq.yml:37]: Dupe ⬆️

File: _data/faq.yml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+  - q: Can services still identify me through other methods?
+    a: Yes, this is just one piece of the puzzle. There are multiple ways services track you, like browser fingerprinting, malicious javascripts, etc... And if you log into services, such as your Google account, they will mostly still know it is you, even though you protect your IP address.
+  - q: Does the SPN only protect my browser?
+    a: The SPN reroutes all your computers connections, including those of your browser.
+  - q: Can I safely log in to sites via the SPN? Like my bank account etc?
+    a: Yes. All serious websites use secure https connections, which encrypts the content of your connection (like login credentials) before the Portmaster wraps them in onion encryption. So no ISP architecture or SPN exit node will be able to manipulate that data. It is protected by https all the way to the destination server.
+  - q: What platforms will you support?
+    a: Since the SPN is integrated into the Portmaster, for our the pre-alpha stage we will only support Windows and Linux. Mac and Mobile are planned, but are further down the road.
```
Comment (dhaavi) [inline _data/faq.yml:30]: Dupe ⬇️

File: _data/faq.yml (hunk 3 above)
Comment (davegson) [reply _data/faq.yml:37]: yes, but they are both important for Portmaster and SPN ;)

File: _data/faq.yml (hunk 2 above)
Comment (davegson) [reply _data/faq.yml:28]: it's gonna be easy to update them as soon as we do.

File: _data/faq.yml (hunk 1 above)
Comment (davegson) [reply _data/faq.yml:24]: Yeah, we also have this set of questions on the SPN right? I think when you're a user you first read the question, and if it interests you, you read on. I want to tackle multiple people, and it usually does not hurt saying the same thing twice or thrice

**Pull request merged**